src/main/golo/decorators.golo

```diff
@@ -150,7 +150,7 @@ function lengthIs = |l| -> |v| {
 
 # .. Checkers .. #
 
-#XXX: checkers defined here need to be functions returning the actuall
+#XXX: checkers defined here need to be functions returning the actual
```

Comment:
Perhaps replace `XXX` :-)

Reply:
This is a classical comment flag (see e.g. in highlight.js), but OK